skbio/stats/tests/test_composition.py

```diff
@@ -1486,8 +1487,7 @@ def test_dirmult_ttest_missing_values_in_table(self):
     def test_dirmult_ttest_inconsistent_indexes(self):
         self.table.index = ['a', 'b', 'c', 'd', 'e']  # Change table index
         with self.assertRaises(ValueError):
-            dirmult_ttest(self.table, self.grouping, self.treatment, self.reference)
-
+            dirmult_ttest(self.table, self.grouping, self.treatment, self.reference)   
```

Comment:
I don't see a test for dirmult_lme, let's make sure to add in multiple tests for this, namely the following

- [ ] Test with the original data from dirmult_ttest, but with an additional column in the metadata with at least 3 subjects (it can be random vector with values [0, 1, 2])
- [ ] Test failure modes to make sure that the input types are checked (i.e. numpy / pandas)
- [ ] Test the reml bool input
- [ ] Test a real dataset (i.e. [moving pictures](https://github.com/qiime2/docs/blob/master/source/tutorials/moving-pictures.rst) to make sure that time, body site and host_subject_id effects can be characterized (body site and host_subject_id should have pvalues that are significant)
- [ ] Add additional tests for including the `re_formula` and possibly the `vc_formula` that are inputted into the statsmodels [mixedlm function](https://www.statsmodels.org/devel/generated/statsmodels.regression.mixed_linear_model.MixedLM.from_formula.html#statsmodels.regression.mixed_linear_model.MixedLM.from_formula). It may require a bit of digging, but I would start with [one of their tutorials](https://www.statsmodels.org/devel/examples/notebooks/generated/mixed_lm_example.html) to see how they use this. 
- [ ] We may need to simulate an example to stress test this. See [this function](https://github.com/biocore/songbird/blob/master/songbird/util.py#L11) for an example of fitting a multinomial GLM (it will require a minor modification to add in random effects, but it should be just a couple of additional lines of code).